shared/src/helpers/permissions/check-has-permission/check-has-permission.helper.ts

```diff
@@ -10,10 +10,14 @@ const checkHasPermission = ({
   permissionKeys,
   userPermissions,
 }: Args): boolean => {
+  const hasPermissions = permissionKeys.length;
```

Comment:
maybe it would sound more clear
```suggestion
  const isRequiringPermissions = Boolean(permissionKeys.length);
```